docs/documentation.rst

```diff
@@ -443,10 +443,10 @@ Contributors
 ------------
 
 A list of contributors for the analysis pipeline is available `here <https://github.com/spine-generic/spine-generic/graphs/contributors>`__.
-If you would like to contribute to this project, please see our `contribution guidelines <../../CONTRIBUTING.md>`_.
+If you would like to contribute to this project, please see our `contribution guidelines <https://github.com/spine-generic/spine-generic/blob/master/CONTRIBUTING.md>`_.
```

Comment:
The point of relative links is that they are not sensitive to change/renaming of repos or organizations. I’d prefer to use them. I think that rst offers that possibility

Reply:
I understand. I have changed links back to relative in https://github.com/spine-generic/spine-generic/pull/102/commits/349a2997dfd6ed022088ec2627b6354aacce0a0d, to `../CONTRIBUTING.md` and `../LICENSE` (so, only `../` instead of `../../`). (I have just tested it in my fork and it could work)

Reply:
> I have just tested it in my fork and it could work

you mean you built the doc locally?

Reply:
@valosekj currently building your branch on RTD-- will add the link when ready

Reply:
@valosekj still broken with 349a2997: https://spine-generic.readthedocs.io/en/jv-100-broken_relative_link_fix/
the doc will build with each push to your branch so you can verify if it works

Reply:
> > I have just tested it in my fork and it could work
> 
> you mean you built the doc locally?

I meant only on my forked `documentation.rst` file: https://github.com/valosekj/spine-generic/blob/master/docs/documentation.rst, not whole doc

Reply:
you need to look at the generated htmls, not the raw rst-- either on RTD (i've added the build for your branch) or locally-- there is a mechanism for sphinx to use relative links-- no time to look into it but just do a quick google and you will find the answer

Reply:
I am just checking it. Exactly how you write, behavior of raw rst and generated html is different.

Reply:
I searched through internet and did a few tests. Sphinx allows relative links, but referred URL must be relative to docs public URL, not to GitHub repo.
I see 3 possible workaround:

1. easiest one - switch from relative link to absolute (`documentation.rst` file already contains several absolute links). CONS - this workaround is sensitive to renaming repo name, organization etc.

2. use `:download:` role how I tried [here](https://github.com/spine-generic/spine-generic/pull/102/commits/769b018e876d8922d1dc1baa27aa97f512162ff3). This workaround however do not forward you to GitHub page after click on the link but download the file to your computer.
I also tried `:download:` role in the latest [commit](https://github.com/spine-generic/spine-generic/commit/8ae38f6e9f713da7f7832693a974390c42b0367e) based on this [recommendation](https://groups.google.com/forum/#!topic/sphinx-users/Aa176x-jVDU) however unsuccessfully.

3. include whole `CONTRIBUTING.md` file directly into html page using `.. include:: ../CONTRIBUTING.md` how I tried [here](https://github.com/spine-generic/spine-generic/pull/102/commits/676846df3533040332e639b594c7aa4502b90136) similarly as `LICENSE`

Reply:
let's go for 1